api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1036,8 +1036,8 @@ public void saveOrder_shouldNotAllowEditingAnExistingOrder() {
 	 */
 	@Test
 	public void getCareSettingByUuid_shouldReturnTheCareSettingWithTheSpecifiedUuid() {
```

Comment:
Why has this test been changed?  It doesn't seem related to the changes being made, and unclear why this is needed.